ensembl/conf/json/ovis_aries_vcf.json

```diff
@@ -3,9 +3,9 @@
     {
       "id": "nextgen_sheep_iroa",
```

Comment:
advisable to update the id to `nextgen_sheep_rambouillet_iroa`

same as [ensembl-variation](https://github.com/Ensembl/ensembl-variation/blob/b7c2637ef23cf37aebe199266088918949674d94/modules/Bio/EnsEMBL/Variation/DBSQL/vcf_config.json#L660)

The id should be unique, it might not make an issue if only this json is loaded but it is better to be safe.